services/movie_session.py

```diff
@@ -0,0 +1,49 @@
+from django.db.models import QuerySet
+from datetime import datetime
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id,
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet[MovieSession]:
+    if session_date:
+        return MovieSession.objects.filter(
+            show_time__date=datetime.strptime(session_date, "%Y-%m-%d")
+        )
+
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: datetime = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None,
+) -> None:
+    movie_session = MovieSession.objects.get(id=session_id)

```

Comment:
Similar to the previous comment, consider handling the case where a `MovieSession` with the given `session_id` does not exist. This can raise a `DoesNotExist` exception.